app/controllers/application_controller.rb

```diff
@@ -83,4 +88,9 @@ def add_token_to_connection
 
     Thread.current[:manage_courses_backend_token] = token
   end
+
+  def assign_sentry_contexts
+    Raven.user_context(id: current_user['user_id'])
+    Raven.tags_context(sign_in_user_id: current_user.fetch('uid'))
```

Comment:
Should probably use safe navigation here since I'm pretty sure `current_user` could be `nil`.

Reply:
I haven't seen how yet ... considering `assign_sentry_contexts` is called in both places after `session[:auth_user]` is set. Can we test this?